WordPress/src/main/java/org/wordpress/android/support/ZendeskHelper.kt

```diff
@@ -162,17 +159,6 @@ private fun configureZendesk(
     )
 }
 
-private fun zendeskFeedbackConfiguration(allSites: List<SiteModel>?, origin: Origin?, extraTags: List<String>?) =
-        object : BaseZendeskFeedbackConfiguration() {
-            override fun getRequestSubject(): String {
-                return ZendeskConstants.ticketSubject
-            }
-
-            override fun getTags(): MutableList<String> {
-                return zendeskTags(allSites, origin ?: Origin.UNKNOWN, extraTags) as MutableList<String>
-            }
-        }
-
 private fun zendeskIdentity(email: String?, name: String?): Identity =
```

Comment:
Same as above - I don't think I understand the issue of the naming here as this one literally creates and returns a Zendesk identity. We can't call it `getZendeskIdentity` because that'll mean that we have an identity to return and not create it. If you think it's better we can call it `createZendeskIdentity`.

Also, I think it'd be more productive and faster if you suggest a naming at least to explain what you find wrong with it.

Reply:
Yes, `createZendeskIdentity()` is what I had in mind.